webapp/src/main/java/org/opentestsystem/rdw/admin/model/CsvSubject.java

```diff
@@ -0,0 +1,19 @@
+package org.opentestsystem.rdw.admin.model;
+
+/**
+ * This class represents the Group subjects allowed during CSV Group uploads.
+ */
+public enum CsvSubject {
```

Comment:
Hmm, i'd lean more towards a parseSubject helper method that knows that `All` == `` == null.